### ISSUE TITLE
Host name vs computer name

### ISSUE THREAD

**Issue by dcbrown16** [MicrosoftDocs/architecture-center#1802]
(I would make an edit but the Edit link is returning a 404)
The "host name" referred to in the note here appears to be what is referred to as "computer name" in the Portal and at the link below.
https://docs.microsoft.com/en-us/azure/virtual-machines/windows/overview#naming

If possible I would like to standardize by changing this doc to use "computer name".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1ddc35bf-00d9-67c8-bf6a-ade5ebea07ca
* Version Independent ID: 7324a1e6-7010-c620-145b-3d8027709545
* Content: [Naming rules and restrictions for Azure resources](https://docs.microsoft.com/en-us/azure/architecture/best-practices/resource-naming)
* Content Source: [docs/best-practices/resource-naming.md](https://github.com/microsoftdocs/architecture-center/blob/master/docs/best-practices/resource-naming.md)
* Service: **architecture-center**
* Sub-service: **cloud-fundamentals**
* GitHub Login: @DerMar
* Microsoft Alias: **pnp**

**Comment by KetanChawda-MSFT** [comment]
@dcbrown16 Thank you for your feedback! We will review and provide an update as appropriate.

**Comment by KetanChawda-MSFT** [comment]
@MikeWasson Any thought here ?

**Comment by absheik** [comment]
AB#160244 - Thanks for reporting - this issue is under review

**Comment by MikeWasson** [comment]
I don't see the phrase "computer name" in the linked article. (It might have changed since you opened the issue back in October) 

Portal uses the phrase "guest host name" which is what this doc is referring to:

![portal](https://user-images.githubusercontent.com/3992422/71920141-fc5c6800-313a-11ea-8095-0e108886f6db.png)

**Comment by MikeWasson** [comment]
Closing because I think there's no inconsistency here.

**Comment by dcbrown16** [comment]
@MikeWasson it does seem to have been removed from the particular doc I linked but I guess the Portal is also inconsistent.
Here's a FAQ entry using "computer name".
https://docs.microsoft.com/en-us/azure/virtual-machines/windows/faq#are-there-any-computer-name-requirements
Here's where it says "Computer name" in the Portal.
![image](https://user-images.githubusercontent.com/34466884/71936836-6dae1200-315f-11ea-9b7f-015e182174cb.png)
Would be nice if we could standardize one way or the other!
